.github/workflows/run-tests.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           restore-keys: ${{ runner.os }}-node-
       - name: Parse Cypress version
         id: parse-cypress-version
-        run: echo "::set-output name=CYPRESS_VERSION::$(jq -r '.devDependencies.cypress' package.json)"
+        run: echo "CYPRESS_VERSION=$(jq -r '.devDependencies.cypress' package.json)" >> $GITHUB_STATE
```

Comment:
I just noticed that it doesn't look like this is working. Prior to this change, in the "Cache Cypress" step the following was logged:
> Cache restored from key: Linux-cypress-11.2.0

But now it is
> Cache restored from key: Linux-cypress-

so doesn't seem to be getting the version

Is it because we're putting it in `GITHUB_STATE` but trying to get it out of the outputs?
```key: ${{ runner.os }}-cypress-${{ steps.parse-cypress-version.outputs.CYPRESS_VERSION }}```

Reply:
Good spot @stephencdaly - I'll raise a PR to fix

Reply:
https://github.com/alphagov/pay-selfservice/pull/3797